extension/test/data_test.ts

```diff
@@ -69,4 +69,60 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    it('should return the text for a word dictionary entry', function () {
```

Comment:
`text` is a bit vague since it's hard for a reader to know the test is correct just from reading the test description and test body.

In a perfect world, I would write this test such that different parts of of the text output were tested independently so if a bug in one part of the text generation failed I could immediately see exactly which part failed due to which test failed.

In this case, the it serves as a pretty good regression test to help ensure no backsliding happens but isn't as good documentation.

Feel free to not change the format but keep in mind the larger philosphy going forward (more important for new code)

Reply:
Thanks for the tip. I agree. The tests should be organized so that when one fails, you have a good idea of what failed just by reading the test description. I'll improve this.